EditorExtensions/Resources/server/services/srv-scss.js

```diff
@@ -35,8 +35,9 @@ var handleSass = function(writer, params) {
             writer.end();
         }
 
-        var css = result.css.toString();
-        var map = result.map.toString();
+        var map, css = result.css.toString();
+        if (!options.omitSourceMapUrl)
+            map = result.map.toString();
```

Comment:
@am11 This has nothing to do with autoprefixer being able to update the existing map. I used the flag because it indicates if source map generation is requested or not ( `omitSourceMapUrl` equals `params.sourceMapURL === undefined`).

 If source maps generation is always enabled, then why its conditional in both autoprefixer and rtlcss ?